src/floating-focus.js

```diff
@@ -226,4 +235,9 @@ export default class FloatingFocus {
 	repositionElement(target, floater) {
 		Object.assign(floater.style, this.getFloaterPosition(target));
 	}
+
+	remove(){
+		this.removeEventListeners()
+		this.floater && this.floater.remove()
```

Comment:
Please use the same formatting as in the rest of the file :) 
(yeah we should have like a prettier linting but whatever for now)
```suggestion
	remove() {
		this.removeEventListeners();
		this.floater && this.floater.remove();
```